react/react-training/src/app/screens/Game/index.js

```diff
@@ -61,4 +51,30 @@ class Game extends Component {
   }
 }
 
-export default Game;
+Game.propTypes = {
+  history: PropTypes.arrayOf(
+    PropTypes.shape({
+      squares: PropTypes.arrayOf(PropTypes.string)
+    })
+  ).isRequired,
+  stepNumber: PropTypes.number.isRequired,
+  xIsNext: PropTypes.bool.isRequired,
+  makeMove: PropTypes.func.isRequired,
+  makeJump: PropTypes.func.isRequired
+};
+
+const mapStateToProps = ({ GameReducer }) => ({
+  history: GameReducer.history,
+  stepNumber: GameReducer.stepNumber,
+  xIsNext: GameReducer.xIsNext
+});
+
+const mapDispatchToProps = dispatch => ({
+  makeMove: (history, squares) => dispatch(Actions.makeMove(history, squares)),
+  makeJump: stepNumber => dispatch(Actions.makeJump(stepNumber))
+});
+
+export default connect(
```

Comment:
Split this component into a smart and dumb component. Check this post: https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0
https://medium.com/@learnreact/container-components-c0e67432e005